ext/session/session.stub.php

```diff
@@ -43,35 +43,45 @@ function session_commit(): bool {}
  * @param callable|object $open
  * @param callable|bool $close
  */
-function session_set_save_handler($open, $close = UNKNOWN, callable $read = UNKNOWN, callable $write = UNKNOWN, callable $destroy = UNKNOWN, callable $gc = UNKNOWN, callable $create_sid = UNKNOWN, callable $validate_sid = UNKNOWN, callable $update_timestamp = UNKNOWN): bool {}
+function session_set_save_handler(
```

Comment:
I was a bit hesitant here. First, I'm not sure how to call the first 2, overloaded params (we could use a construct similar to `$lifetime_or_options`). Besides, we could use the `_handler` prefix: `$read_handler`, `$write_handler` etc.